packages/apex-node/test/tests/asyncTests.test.ts

```diff
@@ -584,6 +580,16 @@ describe('Run Apex tests asynchronously', () => {
       records: codeCoverageQueryResult
     } as ApexCodeCoverageAggregate);
 
+    mockToolingAutoQuery.onCall(4).resolves({
+      done: true,
+      totalSize: 1,
+      records: [
+        {
+          PercentCovered: '57'
+        }
+      ]
+    } as ApexOrgWideCoverage);
```

Comment:
With the change to the use query instead of authFetchQuery some of the stubs needed to be moved around.